### PR TITLE
CI: enable deblike_minion tests

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
## What does this PR change?

Enable deblike minion tests.

## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed
- [X] **DONE**

## Test coverage
- No tests
- [X] **DONE**

## Links

Issue(s):  https://github.com/SUSE/spacewalk/issues/23593

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [X] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Comment by jordimassaguerpla** [comment]
Tests passed. Retriggering to see if this is stable.

**Comment by jordimassaguerpla** [comment]
Tests passed. Retriggering to see if this is stable.

**Comment by jordimassaguerpla** [comment]
Tests passed. We can consider this stable

**Comment by jordimassaguerpla** [comment]
blocked since merging is blocked

**Review by NamelessOne91 — APPROVED** [review]

**Comment by jordimassaguerpla** [comment]
Tests took 33 minutes, same as before.

File: testsuite/podman_runner/17_run_init_clients_tests.sh [hunk 1]
```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 set -xe
-sudo -i podman exec controller-test bash -c "export CUCUMBER_PUBLISH_TOKEN=${CUCUMBER_PUBLISH_TOKEN} && export PROVIDER=podman && export SERVER=uyuni-server-all-in-one-test && export HOSTNAME=controller-test && export SSH_MINION=opensusessh && export MINION=sle_minion && export RHLIKE_MINION=rhlike_minion && cd /testsuite && rake cucumber:github_validation_init_clients"
+sudo -i podman exec controller-test bash -c "export CUCUMBER_PUBLISH_TOKEN=${CUCUMBER_PUBLISH_TOKEN} && export PROVIDER=podman && export SERVER=uyuni-server-all-in-one-test && export HOSTNAME=controller-test && export SSH_MINION=opensusessh && export MINION=sle_minion && export RHLIKE_MINION=rhlike_minion && export DEBLIKE_MINION=deblike_minion && cd /testsuite && rake cucumber:github_validation_init_clients"
```
Comment (srbarrios) [inline testsuite/podman_runner/17_run_init_clients_tests.sh:3]: We need to add this variable in all the rest of scripts.

image

Comment (jordimassaguerpla) [reply testsuite/podman_runner/17_run_init_clients_tests.sh:3]: good point! thanks!

**Comment by jordimassaguerpla** [comment]
This adds 153 new test steps, which means increasing coverage with 1.8%

**Comment by jordimassaguerpla** [comment]
rerunning all jobs to check is they are stable

**Comment by jordimassaguerpla** [comment]
tests pass again. This is stable

**Review by srbarrios — APPROVED** [review]

**Comment by jordimassaguerpla** [comment]
Superseded by https://github.com/uyuni-project/uyuni/pull/8929

**Pull request closed without merging**